assets/scss/base.scss

```diff
@@ -251,6 +251,17 @@ ul.messages {
   }
 }
 
+.input-text { // override if skeleton is being weird
```

Comment:
What about input[name="email"]?

Reply:
Right now there is a bug/something somewhere where the deployed site is ignoring input[name='email'], so putting the styles as an additional class is a half-way solution until that gets fixed. 👍